nautobot/core/views/generic.py

```diff
@@ -163,7 +163,7 @@ def get_filter_params(self, request):
         params = request.GET.copy()
         filter_params = get_filterable_params_from_filter_params(
             params,
-            self.non_filter_params,
+            NON_FILTER_PARAMS,
```

Comment:
Not sure I understand this change. Is `self.non_filter_params` unused now? Have we lost the ability for a specific view to extend this list with its own particular special case parameters?

Reply:
Yeah I did not think about the fact there might be views extending this attribute, I will revert this change